### ISSUE TITLE
Intermittent error uploading symbols with HTTP 403

### ISSUE THREAD

**Issue by bruno-garcia** [getsentry/sentry-javascript-bundler-plugins#605]
### Description

Related to anther issue but that has many different reasons/root causes:
* https://github.com/getsentry/sentry-react-native/issues/3551

This issue was raised on Twitter: https://x.com/thepanta82/status/1833547631521632685
See: https://x.com/thepanta82/status/1833562947517616622

**Comment by panta82** [comment]
Thanks @szokeasaurusrex , that makes sense. I'll try setting up `SENTRY_ALLOW_FAILURE`.

I can't hand-pick IP ranges, because the tooling automatically leases VPS-s on demand, runs jobs, then releases them. Maybe proxy outgoing traffic somehow, but probably not worth the trouble unless the issue escalates.

**Comment by szokeasaurusrex** [comment]
> I can't hand-pick IP ranges, because the tooling automatically leases VPS-s on demand

Gotcha, I was thinking this might be the case given that you are only seeing the errors intermittently. Perhaps there is some way you could manually try making an HTTP request to some Sentry endpoint at the beginning of CI pipeline when the VPS starts, and if you get a `403`, you could immediately retry with a new VPS?

In any case, using `SENTRY_ALLOW_FAILURE` should at least hopefully address your original complaint from [your Tweet](https://x.com/thepanta82/status/1833547631521632685) about wanting a way to avoid having these failures block your CI.

Since we sadly cannot do anything on our end to get these Hetzner IPs unblocked, and since the `SENTRY_ALLOW_FAILURE` at least should unblock your CI, I am going to close this issue for now. Feel free though to reach out with a new issue if you encounter any further difficulties; as long as it is something we are able to assist with, we are happy to help.
